Automattic/wp-calypso
# Home: Update mobile card to display RTL logo when RTL locale is detected

#### Proposed Changes

The update to the mobile card in https://github.com/Automattic/wp-calypso/pull/68062 includes the new "WP → JP" logo. By default, when viewed in RTL, this logo doesn't flow as expected for RTL locales. With this PR, the logo is updated and 'flipped' for RTL users when it's detected that an RTL locale is in use.

#### Testing Instructions

* Apply patch and verify that the app's new "WP → JP" logo display as expected in mobile card found in the customer home sidebar.
* Head to _Me_ → _Account Settings_ to change the account to use an RTL language, such as Hebrew or Arabic.
* Navigate back to the customer home and verify that the "WP → JP" logo is now 'flipped' as expected for RTL users.

**LTR:**

<img width="1149" alt="Screenshot 2022-11-25 at 15 24 21" src="https://user-images.githubusercontent.com/2998162/204016694-bdeaf290-b57b-4eb7-a7cc-af3dfda5ee49.png">

**RTL:**

<img width="1080" alt="Screenshot 2022-11-25 at 15 23 49" src="https://user-images.githubusercontent.com/2998162/204016760-e330c15b-7171-4aac-a1e0-7333385cd96b.png">

#### Pre-merge Checklist
